scripts/sct_analyze_lesion.py

```diff
@@ -27,53 +27,65 @@
-                                 '\n\nIf a registered template is provided, it computes:'
-                                 '\n- the distribution of each lesion depending on each vertebral level and on each region of the template (eg GM, WM, WM tracts).'
-                                 '\n- the proportion of ROI (eg vertebral level, GM, WM) occupied by lesion.'
-                                 '\nN.B. If the proportion of lesion in each region (e.g., WM and GM) does not sum up to 100%, it means that the registered template does not fully cover the lesion, in that case you might want to check the registration results.')
-    parser.add_option(name="-m",
-                      type_value="file",
-                      description="Lesion mask to analyze",
-                      mandatory=True,
-                      example='t2_lesion.nii.gz')
-    parser.add_option(name="-s",
-                      type_value="file",
-                      description="Spinal cord centerline or segmentation file, which will be used to correct morphometric measures with cord angle with respect to slice.",
-                      mandatory=False,
-                      example='t2_seg.nii.gz')
-    parser.add_option(name="-i",
-                      type_value="file",
-                      description="Image from which to extract average values within lesions (e.g. T2w or T1w image, texture image).",
-                      mandatory=False,
-                      example='t2.nii.gz')
-    parser.add_option(name="-f",
-                      type_value="str",
-                      description="Path to folder containing the atlas/template registered to the anatomical image.",
-                      mandatory=False,
-                      example="./label")
-    parser.add_option(name="-ofolder",
-                      type_value="folder_creation",
-                      description="Output folder",
-                      mandatory=False,
-                      example='./')
-    parser.add_option(name="-r",
-                      type_value="multiple_choice",
-                      description="Remove temporary files.",
-                      mandatory=False,
-                      default_value='1',
-                      example=['0', '1'])
-    parser.add_option(name="-v",
-                      type_value='multiple_choice',
-                      description="Verbose: 0 = nothing, 1 = classic, 2 = expended",
-                      mandatory=False,
-                      example=['0', '1', '2'],
-                      default_value='1')
+
+    parser = argparse.ArgumentParser(
+        description='Compute statistics on lesions of the input binary file (1 for lesion, 0 for background). The function assigns an ID value to each lesion (1, 2, 3, etc.) and outputs morphometric measures for each lesion:'
```

Comment:
@YangHee-Min the visual output can be improved. I am working on it (on your branch, for this file)